### ISSUE TITLE
Scream audio clipping

### ISSUE THREAD

**Issue by meatbee** [OpenRCT2/OpenRCT2#7619]
**OS:** Windows 7
**Version:** 0.1.3.
**Commit/Build:** debb6cd

Scream sounds severely clip when 4 or more are played at once (e.g. duelling coasters)

- [ ] Specific to multiplayer?
Don't know, but happened in multiplayer definitely - couldn't save file to disk.
**Steps to reproduce:**
1. View 4x dueling coasters with other coasters simultaneously

**Screenshots / Video:**

https://streamable.com/pd2vl
**Save game:**

Can't get it to appear in folder, but happening on "Fernando's server"/"Feel free to build a park together" currently.

**Comment by ipatix** [comment]
I think this even is an issue with the original game. Probably because the volume is too loud. Could be fixed by adding a limiter to the master audio but I don't know if that would be something appropriate to do.

**Comment by meatbee** [comment]
I put the volume sliders down to 25% and that solved it. I think the default volume is way too high - it only takes 3/4 screams nearby to each other to make loads of terrible clipping happen.

**Comment by live627** [comment]
But then path building audio cues seem way to quiet, even without monkeying with the audio levels.

**Comment by ipatix** [comment]
So is the scream sound generally too loud?

**Comment by live627** [comment]
I think so. But that also holds for the originals. Maybe Chris Sawyer did it on purpose since screams IRL are very loud.

**Comment by meatbee** [comment]
Yeah but the volume balance is a (slightly) separate issue. No audio in a game should be loud enough to clip/distort.

**Comment by ipatix** [comment]
You can't really prevent that unless you limit the amount of simultanous sounds.
